### DISCUSSION TITLE
Issue with Artisan Command loadBackground

### DISCUSSION THREAD

**Discussion by hermetheuscoffee** [artisan-roaster-scope/artisan#747] [Q&A] [answered]
I'm trying to create an Artisan button in v2.4.6 build 2 that will automatically load a background roast profile named "sumatra.alog". I'm quite familiar with buttons in Artisan, and for some reason, I can't get this simple command to work. At this point, I'm thinking it could be a bug. 
The command I'm using is:
`loadBackground(C:\Users\Zoe\Documents\Hermetheus\Profiles\sumatra.alog)`
I've also attached a screenshot of my Events>Buttons screen with the button parameters (ignore that my screenshot shows / instead of \. I have since corrected that, but the issue persists).

![artisan_btn_config](https://user-images.githubusercontent.com/95500043/144658227-520fb416-e340-4629-b663-8dca70738681.png)

When I go to try it out, the app doesn't seem to log ANYTHING. (I have the "Messages" modal window up.) I can successfully use and log the "clearBackground" command. I would expect "loadBackground" to be exactly the same. 

![artisan_btn_notworking](https://user-images.githubusercontent.com/95500043/144658228-931e241a-67fe-4022-94cb-c3c8036864bf.png)

I've also tried copying my sumatra.alog file directly to the root of  C:\ and tried it from there, and it also does nothing. I've also tried "loadbackground" (lowercase "b"), since these are case-sensitive. For the life of me, I can't get loadBackground to work. 

Can someone else confirm this or offer guidance?

**Comment by MAKOMO** [comment, marked as answer]
Does it work to put the path in double quotes?

`loadBackground("C:\Users\Zoe\Documents\Hermetheus\Profiles\sumatra.alog")`

**Reply by hermetheuscoffee** [reply]
Yep, that was it!! I felt like I tried every permutation of the file path, but I hadn't considered double quotes. Thanks for the fast solution, Marko! (And for the follow on Instagram.) :)

**Reply by MAKOMO** [reply]
Just added some code to allow a path without double quotes here too. Will be released with the next version.  Thanks for testing, Marko

**Reply by bigl00z3** [reply]
Never realised this was there. Just adding some buttons for quick background loading.
Thanks guys.
